test/test_euler.py

```diff
@@ -431,9 +469,10 @@ def test_facial_flux(actx_factory, order, dim):
     )
 
 
+@pytest.mark.parametrize("nspecies", [1, 2, 3])
 @pytest.mark.parametrize("dim", [1, 2, 3])
 @pytest.mark.parametrize("order", [1, 2, 3])
```

Comment:
These are 27 tests. That's way too many. Use tuple parametrization to avoid the full Cartesian product.